lib/src/tzdata/zone_tab.dart

```diff
@@ -56,7 +56,7 @@ class LocationDescriptionDatabase {
   LocationDescriptionDatabase(this.locations);
 
   factory LocationDescriptionDatabase.fromString(String data) {
-    final lines = data.split('\n');
```

Comment:
Is this necessary given the fix in `tool/encode_tzf.dart`?

Reply:
Yes, this change modified the line ends from windows convention (CR+LF='\r\n') to Unix (LF only='\n'), before the split on LF only. Without this line there are CR remaining in the strings.

The change in encode_tzf.dart changes backward slashes '' with forward slashes '/' in the names of the timezones. (The backward slashes are coming form the filepaths on windows.)

Reply:
Ugh, sorry, brain fart when I reviewed this. Thanks!